app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/forms/TestCreateJourneyForm.scala

```diff
@@ -117,7 +119,8 @@ object TestCreateJourneyForm {
         journeyConfig.pageConfig.signOutUrl,
         journeyConfig.pageConfig.accessibilityUrl,
         journeyConfig.businessVerificationCheck,
-        journeyConfig.regime
+        journeyConfig.regime,
+        if(journeyConfig.pageConfig.labels.isDefined)Some(journeyConfig.pageConfig.labels.get.optWelshServiceName) else None
```

Comment:
Richard up to you if you want to change t:
 journeyConfig.pageConfig.labels.map(_.optWelshServiceName)
For me is good either way.

Reply:
Good idea. Done.